ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/StreamQueryResponseHandler.java

```diff
@@ -88,6 +106,10 @@ protected void handleRow(final Buffer buff) {
         LOG.info("Response contains consistency vector " + jsonObject);
         serializedConsistencyVector.set((String) ((JsonObject) json).getMap().get(
             "consistencyToken"));
+      } else if (jsonObject.getMap() != null && jsonObject.getMap().containsKey("continuationToken")) {
+        LOG.info("Response contains continuation token " + jsonObject);
+        continuationToken.set((String) ((JsonObject) json).getMap().get(
```

Comment:
We save the continuation token here from the query response

Reply:
this should be another if statement, not an else if, since we may want to save the consistency token and the continuation token

Reply:
I agree, that makes sense.  Not sure in what situation we'd have both (not currently a thing), but maybe it could be.